samples/matter/closure/boards/nrf54l15dk_nrf54l15_cpuapp_internal.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
This file can be removed, we don't need to support internal configuration with nRF54L15